### DISCUSSION TITLE
not able to install on rasberry pi

### DISCUSSION THREAD

**Discussion by samhanna84** [dani-garcia/vaultwarden#1336] [General]
Status: Downloaded newer image for bitwardenrs/server:raspberry
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm/v7) and no specific platform was requested
31308a5feaea98139de46cc3355b55a7f7eba645a46abbdcfa2bfd59882fb58b

**Comment by samhanna84** [comment]
I'm running it on a raspberry pi 3

**Comment by samhanna84** [comment]
I've also tried it with the latest tag and I still get the same issue

**Comment by SrGato001** [comment]
use: bitwardenrs/server:latest-arm32v6
